workflows/explore_eml/access_specific_elements.Rmd

```diff
@@ -33,10 +33,28 @@ eml_get(doc, "url")
 '': ecogrid://knb/urn:uuid:89bec5d0-26db-48ac-ae54-e1b4c999c456
 ```
 
-You can also use the `which_in_eml()` function from the `datamgmt` package to get indices within an EML list. Here is an example:
+`eml_get_simple()` is a simplified alternative to `eml_get()` that produces a list of the desired EML element.
+
+```{r "eml_get_simple", eval = FALSE}
+eml_get_simple(doc$dataset$otherEntity, "entityName")
+```
+
+To find an eml element you can use either a combination of `which_in_eml`from the `datamgmt` package or `eml_get_simple` and `which` to find the index in an EML list. Use which ever workflow you see 
```

Comment:
I think `which_in_eml` moved to `arcticdatautils`

Reply:
updated thanks!